modules/@apostrophecms/schema/index.js

```diff
@@ -2247,7 +2263,7 @@ module.exports = {
           self.apos.util.error(format(s));
         }
         function format(s) {
-          return stripIndent`
+          return stripIndents`
```

Comment:
I changed to stripIndents because stripIndent only catches the first line, which is not the intent when looking at the pattern below.